clang/lib/Frontend/CompilerInstance.cpp

```diff
@@ -547,10 +547,25 @@ void CompilerInstance::createPreprocessor(TranslationUnitKind TUKind) {
     PP->setDependencyDirectivesGetter(*GetDependencyDirectives);
 }
 
+SmallString<256>
```

Comment:
It's usually preferable to use a `SmallVectorImpl<char> &` out parameter so the caller can decide the inline storage size